src/main/java/com/marek/onlinebookstore/controller/OrderController.java

```diff
@@ -0,0 +1,70 @@
+package com.marek.onlinebookstore.controller;
+
+import com.marek.onlinebookstore.dto.order.OrderDto;
+import com.marek.onlinebookstore.dto.order.OrderItemDto;
+import com.marek.onlinebookstore.dto.order.PlacingOrderRequestDto;
+import com.marek.onlinebookstore.model.User;
+import com.marek.onlinebookstore.service.order.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
```

Comment:
Consider implementing exception handling to handle possible errors during the process of making an order, getting an order history or getting items from an order. This can improve the robustness of your application.